source/release-notes.html.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Release notes
+weight: 200
+---
+
+# Release notes
+
+### Release 0.2 - 11 September 2019
+
+- Limit the number of [offer conditions](/make-an-offer/#attributes) to 20
```

Comment:
Should we include "Remove interviews"? https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training-tech-docs/commit/08c7cda1b920ef2d23f37dfeeff338eb965415b6

Reply:
Added!